ethers-providers/src/provider.rs

```diff
@@ -164,6 +164,40 @@ impl<P: JsonRpcClient> Provider<P> {
             }
         })
     }
+
+    async fn get_uncle_count_gen(&self, id: BlockId) -> Result<U256, ProviderError> {
```

Comment:
```suggestion
    async fn get_uncle_count(&self, id: BlockId) -> Result<U256, ProviderError> {
```
Same comment as below